vue-components/src/components/InputWithExtender.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+	<div
+		:class="[ 'wikit', 'wikit-InputWithExtender' ]"
+	>
+		<label class="wikit-InputWithExtender__label" :for="id">{{ label }}</label>
+		<Input
+			:id="id"
+			:value="value"
+			@focus.native="showExtension = true"
+			@blur.native="showExtension = false"
+			@esc.native="showExtension = false"
```

Comment:
That is actually not working, because there is no `@esc` event in vuejs. It needs to use the `@keydown` event with the `.esc` modifier: `@keydown.esc="..."`, see: https://vuejs.org/v2/guide/events.html#Key-Modifiers

Reply:
Done

Reply:
keydown.esc didn't work on its own, needed native too but worked after it